setup.py

```diff
@@ -76,8 +93,32 @@
         include_package_data=True,
         extras_require={
             'alldeps': (
-                'numpy >= {}'.format(NUMPY_MIN_VERSION),
-                'scipy >= {}'.format(SCIPY_MIN_VERSION),
+                f'numpy>={NUMPY_MIN_VERSION}',
+                f'scipy>={SCIPY_MIN_VERSION}',
```

Comment:
Are you sure we need this? I would have though that them being in `install_requires` they would be installed in any case?

Reply:
It is what was thinking but it seems that they were there before. I am not sure the reason.

Reply:
They were there before and I've just adapted the code for the syntax and to add `joblib` and `threadpoolctl`.
But they needn't be there.